README.md

```diff
@@ -0,0 +1,68 @@
+openant
+=======
+
+A python library to download and upload files from ANT-FS compliant devices (Garmin products).
+
+Features
+--------
+
+ - ANT
+ - ANT-FS (with command pipe, file listings, downloading, uploading, etc)
+ - Three libs (ant.base basic ANT library, ant.easy blocking version using ant.base, ant.fs ANT-FS library)
+
+Requirements
+------------
+
+- Python 2.7+
+- PyUSB > 1.0a2 (seems to be a bug which makes it segfaults on version before Alpha 2)
```

Comment:
For python>=3.4 is requires current PyUSB master which has fixed [this issue](https://github.com/walac/pyusb/issues/66), but it still not released. Maybe we could add a note in parenthesis to explain this while waiting for new release. No problem with python 3.3 and python 2.7.